src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -252,21 +271,17 @@ const styles = StyleSheet.create({
   },
   card: {
     height: 64,
-    // height: 150,
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
     flex: 1,
     paddingLeft: 20
-    // backgroundColor: 'red'
-    // marginVertical: 10,
   },
   followContainer: {
-    paddingRight: 20,
-    paddingLeft: 8,
+    paddingRight: normalize(20),
+    paddingLeft: normalize(8),
     height: '100%',
     justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height comment has been removed from the card style. If this comment was important for understanding the code, consider adding it back. The paddingRight and paddingLeft properties in the followContainer style have been wrapped with the normalize function. Ensure that this does not negatively impact the layout of the followContainer in any scenario.


<!-- This is an auto-generated comment by OSS CodeRabbit -->